package.json

```
@@ -13,10 +13,21 @@
   },
   "license": "MIT",
   "author": "Keith Cirkel <npm@keithcirkel.co.uk> (http://keithcirkel.co.uk/)",
+  "type": "module",
+  "exports": {
```

Please add:
```suggestion
  "engines": {
    "node": ">=20"
  },
  "exports": {
```

Additionally please `npx sort-package-json`! :wink: